runtime-attach/src/main/java/io/opentelemetry/contrib/attach/RuntimeAttach.java

```diff
@@ -7,22 +7,82 @@
 
 import java.io.File;
 import java.lang.management.ManagementFactory;
+import java.util.logging.Logger;
 import net.bytebuddy.agent.ByteBuddyAgent;
 
+/** This class allows you to attach the OTel agent for Java at runtime. */
 public final class RuntimeAttach {
 
+  private static final Logger LOGGER = Logger.getLogger(RuntimeAttach.class.getName());
```

Comment:
to match convention in the OTel Java repos
```suggestion
  private static final Logger logger = Logger.getLogger(RuntimeAttach.class.getName());
```

Reply:
I think this comment got collapsed so was missed (this collapsing is my least favorit github feature)